apps/workspaces/actions.py

```
@@ -186,7 +186,7 @@ def get_workspace_admin(workspace_id):
     return admin_email
 
 
-def export_to_xero(workspace_id, export_mode="MANUAL", expense_group_ids=[], is_direct_export:bool = False, triggered_by: ExpenseImportSourceEnum = None):
+def export_to_xero(workspace_id, expense_group_ids=[], is_direct_export:bool = False, triggered_by: ExpenseImportSourceEnum = None):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)